ciao-cli/image.go

```diff
@@ -350,6 +351,87 @@ func uploadTenantImage(username, password, tenant, image, filename string) error
 	return res.Err
 }
 
+type imageModifyCommand struct {
+	Flag       flag.FlagSet
+	name       string
+	imageID    string
```

Comment:
Other commands such as "image upload" have an "image" argument/variable
you might want to change this for consistency